CONTRIBUTING.md

```diff
@@ -13,3 +13,40 @@ in other words it allows to separate the actual name of the variable and the nam
 
 In the case of a pseudo-local variable (a field used briefly to pass around a local variable of a method between 2 injections of said method),
 the field should be named with the namespace first, then the name of the injected method, and finally the name of the local (`quilt$injectedMethod$localName`).
+
+## Licensing
+
+Quilt Standard Libraries are licensed under [Apache 2.0][LICENSE], which means your contributions should not introduce code of incompatible licensing.
```

Comment:
Don't ask me why, but "QSL" needs no definite article and "Quilt Standard Libraries" usually does
```suggestion
QSL is licensed under [Apache 2.0][LICENSE], which means contributions should not introduce incompatibly licensed code.
```